scripts/albs-682-fixes.py

```diff
@@ -215,19 +286,12 @@ async def delete_wrong_packages():
 
 
 async def delete_packages_prefix():
-    logging.info("Removing 'Packages/' prefix from filenames")
-    pulp = pulp_client.PulpClient(
-        settings.pulp_host, settings.pulp_user, settings.pulp_password
-    )
-
+    logging.info("Deleting 'Packages/' prefix from filenames")
     with PulpSession() as pulp_db, pulp_db.begin():
-        advisory_pkgs = pulp_db.execute(
-            select(UpdatePackage)
-        ).scalars().all()
+        advisory_pkgs = pulp_db.execute(select(UpdatePackage)).scalars().all()
 
         for pkg in advisory_pkgs:
             if pkg.filename.startswith("Packages/"):
-                logging.info(f"Removing 'Packages/' prefix from {pkg.filename} filename")
```

Comment:
@anfimovdm, I added this line to get a list of the pkg filenames we are fixing when running the script in prod. IMHO we should keep it.